src/styles/blocks/page.scss

```diff
@@ -0,0 +1,4 @@
+.page__body {
+  margin: 0;
+  font-family: Roboto, sans-serif, system-ui;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), when specifying font-family, avoid using generic family keywords like 'sans-serif' or 'serif' as intermediate fallbacks. Only use them as a last resort. Instead, specify an ordered list of font families for the browser to choose from.